bigchaindb/tendermint/event_stream.py

```diff
@@ -0,0 +1,84 @@
+import json
+import logging
+import time
+
+import asyncio
+import aiohttp
+
+from bigchaindb.common.utils import gen_timestamp
+from bigchaindb.events import EventTypes, Event
+from bigchaindb.tendermint.utils import decode_txn_base64
+
+
+HOST = 'localhost'
+PORT = 46657
+URL = f'ws://{HOST}:{PORT}/websocket'
+
+PAYLOAD = {
+    "method": "subscribe",
+    "jsonrpc": "2.0",
+    "params": ["NewBlock"],
+    "id": "bigchaindb_stream"
+}
+
+
+logger = logging.getLogger(__name__)
+
+
+async def connect_and_recv(event_queue):
```

Comment:
`async` and `await` are not compatible with Python 3.5, that we still support. I'd be happy to drop support for Python 3.5, but it must be something planned.

In case of function definitions and function calls, we can easily use `@asyncio.coroutine` and `yield from`, for `with` and `for` is more difficult.

I've created #1956 to talk about this issue.